### PR TITLE
Added description how user-defined PVCs are processed on different PVC strategies

### PR DESCRIPTION
**Pull request by sleshchenko**
### What does this PR do?
Adds description how user-defined PVCs are processed on different PVC strategies

### What issues does this PR fix or reference?
https://github.com/eclipse/che/pull/12335

### CODE REVIEW

**Comment by skabashnyuk** [comment]
@dneary @slemeur @rkratky can you take a look on this PR?

File: src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc [hunk 1]
```diff
@@ -99,7 +99,7 @@ In the command above, `eclipse-che` is the Che namespace.
 [id="how-the-che-server-uses-PVCs-and-PVs-for-storage"]
 == How the Che server uses PVCs and PVs for storage
 
-Che server, Keycloak and PostgreSQL pods, and workspace pods use Persistent Volume Claims (PVCs), which are bound to the physical Persistent Volumes (PVs) with https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[ReadWriteOnce access mode]. When the deployment YAML files run, they define the Che PVCs. You can configure link:#che-workspaces-storage[workspace PVC] access mode and claim size with Che deployment environment variables.
+Che server, Keycloak and PostgreSQL pods, and workspace pods use Persistent Volume Claims (PVCs), which are bound to the physical Persistent Volumes (PVs) with https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes[ReadWriteOnce access mode]. When the deployment YAML files run, they contain PVCs or define the Che Volumes. You can configure link:#che-workspaces-storage[workspace PVC] access mode and claim size with Che deployment environment variables.
```
Comment (slemeur) [inline src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc]: "When the deployment YAML files run, they contain PVCs or define the Che Volumes."
I don't understand.
- Can a YAML file "run"
- they contain PVCs --> ?
- or define Che Volumes ???

File: src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc [hunk 2]
```diff
@@ -128,9 +128,25 @@ Sub-paths pre-created |Easy to manage and control storage |Workspace containers
 [id="unique-pvc-strategy"]
 === Unique PVC strategy
 
-To define the unique strategy, set `CHE_INFRA_KUBERNETES_PVC_STRATEGY` to `unique`.
+[id="how-the-unique-pvc-strategy-works"]
+==== How the unique PVC strategy works
 
-Every workspace gets its own PVC, which means a workspace PVC is created when a workspace starts for the first time. Workspace PVC is deleted when a corresponding workspace is deleted.
+Every Che Volume of workspace gets its own PVC, which means a workspace PVCs are created when a workspace starts for the first time. Workspace PVCs are deleted when a corresponding workspace is deleted.
```
Comment (slemeur) [inline src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc]: which means workspace PVCs 
remove "a"

**Review by slemeur — CHANGES_REQUESTED** [review]

**Comment by slemeur** [comment]
Also do you confirm it is working the same for OS?

File: src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc (hunk 1 above)
Comment (sleshchenko) [reply src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc]: Good catch. I misunderstood this sentence. Now looks I should just revert my changes here. Will do it.

File: src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc (hunk 2 above)
Comment (sleshchenko) [reply src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc]: done

File: src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc (hunk 1 above)
Comment (sleshchenko) [reply src/main/pages/che-6/setup-kubernetes/kubernetes-or-openshift-admin-guide.adoc]: Reverted changes in this line. Please resolve comment if existing line

> When the deployment YAML files run, they define the Che PVCs. You can configure link:#che-workspaces-storage[workspace PVC] access mode and claim size with Che deployment environment variables.

is OK for you.

**Comment by sleshchenko** [comment]
> Also do you confirm it is working the same for OS?

Yes, it works for both of Kubernetes and OpenShift infrastructures. The modified page is included in `Che on Kubernetes: Admin Guide` and `Che on OpenShift: Admin Guide` articles.
@slemeur I fixed your comments, Could you please review one more time?

**Review by slemeur — APPROVED** [review]

**Comment by slemeur** [comment]
@rkratky is that fine for you?

**Comment by slemeur** [comment]
@rkratky : that PR has slipped while the content is making a great difference. I fixed the conflicts and going to merge, as there are explanation that would be much valuable for Che 7 (and CRW too).

**Pull request merged**